docs/ruleset_functions/zone_meets_G3_1_1e.md

```diff
@@ -0,0 +1,60 @@
+# zone_meets_G3_1_1e
+**Schema Version:** 0.0.22  
+
+**Description:** determines whether a given zone meets the G3_1_1e exception "Thermal zones designed with heating-only systems in the proposed design serving storage rooms, stairwells, vestibules, electrical/mechanical rooms, and restrooms not exhausting or transferring air from mechanically cooled thermal zones in the proposed design shall use system type 9 or 10 in the baseline building design."
+
+**Inputs:**
+- **P-RMR**
+- **B-RMR**
+- **zone**

```

Comment:
Depends on how the function is used 
In this case, If `P-RMD` is an argument, a zone seems unnecessary. A `zone_id` should do well here.

Reply:
in line 21, the function iterates through the spaces in the zone, so makes sense to me to pass a zone, instead of an id:

`for space_p in zone.spaces:`

Reply:
The `P-RMD` contains the zone data. So passing a `zone_id` can locate the zone in the `P-RMD` by calling `zone=find_zone(zone_id)`. 
However, this is a detail implementation. Let's leave it for now. No worries.

Reply:
ah, that's a useful function for me to know!  Wouldn't it be more efficient from a programming perspective to pass a zone (which is just a pointer) rather than an id and use a function to find the zone?

Although the logic works either way & no need to spend time on it, just curious.

Reply:
That is very good question. It makes sense in low-level languages such as C or C++, which uses pointers ( in that case, we should label them as *zone) pointing to a chunk of computer memory that stores the object. However, in python, this makes not much difference in terms code efficiency. Passing a zone ID is probably cleaner since the zone object data is also contained in B_RMD.